README.md

```diff
@@ -174,6 +174,13 @@ embedded.kafka.topicsToCreate=some_topic
   * To use another kafka version pick corresponding docker image on  [dockerhub](https://hub.docker.com/r/confluentinc/cp-kafka/tags)
   * [Confluent Platform and Apache Kafka Compatibility](https://docs.confluent.io/current/installation/versions-interoperability.html#cp-and-apache-kafka-compatibility)
 * `embedded.kafka.waitTimeoutInSeconds` (default is 60 seconds)
+* `embedded.schema-registry.enabled` (true|false, default is 'true')
```

Comment:
let's have ```embedded.kafka.schema-registry.enabled```
since we might have other schema-registry
or 
```embedded.kafka-schema-registry.enabled```

Reply:
+1
also lets make default as false, or move to another module

Reply:
- Changed the prefix to `embedded.kafka.schema-registry`

- Changed schema registry to be disabled by default